app/classifier/tasks/survey/index.cjsx

```diff
@@ -75,15 +78,16 @@ module.exports = React.createClass
     @setState filters: {}
 
   clearSelection: ->
-    @setState 
+    @setState
       selectedChoiceID: ''
       focusedChoice: @state.selectedChoiceID
     newAnnotation = Object.assign {}, @props.annotation, _choiceInProgress: false
     @props.onChange newAnnotation
 
   handleAnnotation: (choice, answers, e) ->
     filters = JSON.parse JSON.stringify @state.filters
-    value = @props.annotation.value?.slice?(0) ? []
+    currentSelection = @state.selectedChoiceID
+    value = @props.annotation.value?.filter (value) -> value.choice isnt currentSelection
```

Comment:
This might not do what you expect if, for some reason, `currentSelection` and `choice` have different values. They should be the same, but I'm not sure about relying on that assumption.